webpack.config.js

```diff
@@ -1,5 +1,17 @@
+const webpack = require("webpack");
```

Comment:
Interesting, why do you need to require webpack in your second commit? It seems like it was not needed for the first one.